.github/workflows/daily.yml

```diff
@@ -32,12 +31,17 @@ jobs:
           - 747000
           - 748000
           - 749000
+        runner:
+          - native
+          - vm
       fail-fast: false
     defaults:
       run:
         shell: bash
         working-directory: ./starknet-replay
 
+    # todo: build first on a single job, and run the blocks after
```

Comment:
Should this work?
I think that since there are different workflows, we can build all just once

Reply:
I saw that in the VM something similar is done: https://github.com/lambdaclass/cairo-vm/blob/159f67da19964cc54a95423a69470a26e534a13d/.github/workflows/rust.yml#L393-L410. I thought it would be nice to add it, although in a 6h workflow, building the binary may be negligible, so maybe is not that worth it

Reply:
ahhh okay, if we save it in cache it could work, but as you said I think it won`t make a significant change
let just remove this todo

Reply:
agreed, I'll remove it